packages/venia-ui/lib/components/OrderHistoryPage/orderHistoryPage.css

```diff
@@ -20,13 +20,21 @@
     row-gap: 1rem;
 }
 
+.filterRow {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.pageInfo {
+    font-size: 0.875rem;
```

Comment:
Is there a token we can use for this? Seems like we'd want consistency across the app for these.

Reply:
Good call, switched to token in 8c923bf.